contracts/extensions/OneTxPayment.sol

```diff
@@ -24,8 +24,8 @@ import "./ColonyExtension.sol";
 
 
 contract OneTxPayment is ColonyExtension {
-  event OneTxPaymentMade();
-  
+  event OneTxPaymentMade(address madeBy);
```

Comment:
I don't like this argument name for some reason -- it seems unconventional. Can we go with `caller` or `agent` or something?